doc/source/flexibility/extended_mysqldump.rst

```diff
@@ -9,11 +9,11 @@ Extended ``mysqldump``
 Backup Locks support
 ====================
 
-In |Percona Server| :rn:`5.7.10-1` ``mysqldump`` has been extended with a
-new option, :option:`lock-for-backup` (disabled by default). When used together
-with the :option:`--single-transaction` option, the option makes ``mysqldump``
-issue ``LOCK TABLES FOR BACKUP`` before starting the dump operation to prevent
-unsafe statements that would normally result in an inconsistent backup.
+In |Percona Server| :rn:`5.7.10-1` ``mysqldump`` was extended with a new option,
```

Comment:
Specific version should not matter in previous release series, more like "mysqldump has a new option ... "